src/app/+dashboard/components/top-chart/top-chart.component.scss

```diff
@@ -0,0 +1,74 @@
+.top {
+    height: 90%;
+    // background-color: bisque;
+
+    .top-header {
+        display: flex;
+        width: 40%;
+        justify-content: space-between;
+        border-bottom: 1px solid #38c215;
```

Comment:
since you have a colors file, where you store colors variables - you can move this color into that file, because it is reusable in multiple places.

Reply:
fixed